src/Mariuzzo/LaravelJsLocalization/Commands/LangJsCommand.php

```diff
@@ -85,6 +85,6 @@ protected function getOptions()
      */
     public function getPublicPath()
     {
-        return $this->laravel['public_path'];
+        return public_path();
```

Comment:
I'm worried about this, because it will break the tests. Does `$this->laravel['public_path']` fails?